src/Moryx.Orders.Management/Implementation/JobHandler/SingleJobOperationDispatcher.cs

```diff
@@ -14,8 +15,13 @@ namespace Moryx.Orders.Management
     [Plugin(LifeCycle.Singleton, typeof(IOperationDispatcher), Name = nameof(SingleJobOperationDispatcher))]
     public class SingleJobOperationDispatcher : OperationDispatcherBase
     {
+        /// <summary>
+        /// ParallelOperations injected by the container
+        /// </summary>
+        public IParallelOperations ParallelOperations { get; set; }
```

Comment:
Introduced but never used

Reply:
Check L55, moved it to this implementation. It was removed from the base because there it was unused.